### PR TITLE
Offer improvement. Remove two useless expressions.

### PR DESCRIPTION
**Pull request by gonaumov**
## Description
There is the following errors into FireFox Web console 
```
TypeError: useless expression hexagon.js:8987:4
TypeError: useless expression hexagon.js:15313:11
```
## Motivation and Context
In fact this code is redundand. I want to help code to be clean and error free. 
## How Has This Been Tested?
I run the tests. I also replace content of hexagon.js file on 
https://www.hexagonjs.io/ with TamperMonkey user script.

## Checklist:

- [ ] I have added a tag to this pull request that indicates the impact of the change (patch, minor or major)
- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly. (This includes updating the changelog).
- [x] I have read the **CONTRIBUTING** document.
- [x] All my changes are covered by tests.
- [x] This request is ready to merge

### CODE REVIEW

File: modules/color-picker/main/index.coffee [hunk 1]
```diff
@@ -227,7 +227,7 @@ class ColorPicker extends hx.EventEmitter
           if endCallback isnt null then endCallback(elem)
           dragging = false
 
-        dispose: ->
+        dispose = ->
```
Comment (jmsmyth) [inline modules/color-picker/main/index.coffee]: It looks like `dispose()` and `disposed` are not used any more so this function can possibly be removed entirely

File: dist/hexagon-dark/hexagon.js [hunk 2]
```diff
@@ -15257,7 +15256,7 @@ ColorPicker = (function(superClass) {
           return circleMoved(pos);
         };
         dragObject = function(elem, parent, min, max, startCallback, moveCallback, endCallback) {
-          var StartListening, StopListening, cursorStartPos, disposed, dragGo, dragStart, dragStop, dragStopHook, dragging, elementStartPos, listening, temp;
```
Comment (jmsmyth) [inline dist/hexagon-dark/hexagon.js]: The `dist/` files don't normally get updated until a release - could these bits be left out the PR?

File: modules/color-picker/main/index.coffee (hunk 1 above)
Comment (gonaumov) [inline modules/color-picker/main/index.coffee]: It seems that disposed is used on 8 places:
```
georginaumov@georgi:~/Documents/hexagonjs$ grep -R disposed --include '*.coffee' *
modules/color-picker/main/index.coffee:        cursorStartPos = elementStartPos = dragging = listening = disposed = null
modules/color-picker/main/index.coffee:          if dragging or not listening or disposed then return
modules/color-picker/main/index.coffee:          if !dragging or disposed then return
modules/color-picker/main/index.coffee:          if not dragging or disposed then return true
modules/color-picker/main/index.coffee:          if disposed then return
modules/color-picker/main/index.coffee:          disposed = true
modules/color-picker/main/index.coffee:          if listening or disposed then return
modules/color-picker/main/index.coffee:          if not listening or disposed then return
georginaumov@georgi:~/Documents/hexagonjs$ grep -R disposed --include '*.coffee' * | wc -l
8
```
but I can remove it. Are you sure? By the way this is my first try with cofeescript so may be I missing something.

Comment (jmsmyth) [inline modules/color-picker/main/index.coffee]: Should be ok I think. The only place `disposed` is assigned to is in the `dispose` function which isn't used, so `disposed` is always going to be null.

Comment (gonaumov) [inline modules/color-picker/main/index.coffee]: This is done. I removed dispose method and disposed variable completely.

File: dist/hexagon-dark/hexagon.js (hunk 2 above)
Comment (gonaumov) [inline dist/hexagon-dark/hexagon.js]: This is done.

**Pull request merged**